c/diff_ci.sh

```diff
@@ -15,6 +15,17 @@ else
   exit
 fi
 
+find_parent_with_Makefile() {
+  while [[ "$parent_with_Makefile" != "." ]]; do
+    if [ -e "$parent_with_Makefile/Makefile" ]; then
+      return 0
+    else
+      parent_with_Makefile=${parent_with_Makefile%/*}
+    fi
+  done
+  echo "No Makefile found in any parent."
+  exit
```

Comment:
Just a suggestion: Wouldn't it be better to return with a failure here?

Reply:
yes, thanks